src/pages/waiting-room-to-car/waiting-room-to-car.selector.ts

```diff
@@ -0,0 +1,5 @@
+import { CategoryCode } from '@dvsa/mes-test-schema/categories/common';
+
+export const displayCabLockDown = (categoryCode: CategoryCode) => categoryCode === 'C' || categoryCode === 'C+E' ;
```

Comment:
I would just add these as functions to the WRTC cat-c page. They are not really selectors.

Reply:
I agree with Szabi, it's more logic rather than selector

Reply:
+ We could also use the test category enum here instead of magic strings

Reply:
We already have enums in TestCategory.ts dependency, could use that value

Reply:
done